docs/markdown/GettingStarted_UI.md

```diff
@@ -0,0 +1,86 @@
+# The Inviwo Graphical Interface
+This part of the Getting Started Guide explains Inviwo's graphical user interface and the rough concepts
+of its building blocks. The goal of this guide is to give an intuitive feeling for the GUI and enable
+users to build workspaces from existing processors.
+
+## GUI Overview
+The following image shows Inviwos GUI:
+
+![The Inviwo GUI](images/UI.png)
+
+As you can see, the main area consists of the *Network Editor*,
+surrounded by the [*Processor*](#markdown-header-processors) *List* on the left, the selected processor's
+[*Properties*](#markdown-header-properties) to the right and the *Debug Console* with log messages below.
+
+You can drag processors from the *Processor List* into the *Network Editor* using drag and drop.
+Processors are the basic building blocks of the network and perform one encapsulated action.
+Each processor can define its required input and produced output data through [*Ports*](#markdown-header-ports),
+which can be connected using the mouse. The resulting network represents an acyclic graph where
+the nodes are processors and the edges are port connections.
+In addition to inputs and outputs, a processor's [*Properties*](#markdown-header-properties) can be edited
+on the right when the processor is selected. Those properties represent the state of the processor and
```

Comment:
avoid "state" we prefer to think of it as configuration parameters or something like that. 
from the perspective of a user of the GUI the network should operate in a pure functional stateless fashion. 
i.e. output = f_i1,i1(g_j1,j2(input1), h_i3(k_i3(input2)))
where i1, i2, i3... are the properties.